etc/kayobe/inventory/group_vars/controllers/network

```diff
@@ -0,0 +1,28 @@
+# List of IP routing tables. Each item should be a dict containing 'id' and
+# 'name' items. These tables will be added to /etc/iproute2/rt_tables.
+network_route_tables:
+  # IP routing table for traffic orginating from the public network. Because of
```

Comment:
Is it more accurate to say, routed from our public IP?  In this specific case, the traffic from eod-seed is originating from 10.12.3.2(IIRC) and is routed to 10.12.9.254.  Our response, from 10.12.9.254, needs to be routed via 10.12.9.1